<!-- -->
<launch>
    <node pkg="can_encoder" type="can_encoder" name="can_encoder" output="screen">
        <param name="auto_speed" type="bool" value="true"/>
        <param name="auto_steer" type="bool" value="false"/>
        <param name="bodywork_control" type="bool" value="false" />
    </node>
</launch>
